ListingManager/ListingManager.cs

```diff
@@ -0,0 +1,306 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+
+namespace ListingUpdater
+{
+    public static class ListingManager
+    {
+        public static ICollection<string> GetAllExtraListings(string pathToStartFrom)
+        {
+            var files = new List<string>();
+            
+            FileManager.GetAllFilesAtPath(pathToStartFrom, true).ToList().ForEach(f =>
+            {
+                if (IsExtraListing(f))
+                {
+                    files.Add(f);
+                }
+            });
+
+            return files;
+        }
+        
+        private static bool IsListing(string listingPath, out ListingInformation listingData)
```

Comment:
This method appears to follow the normal `TryGet*` pattern. It should probably be renamed to reflect this.